### PR TITLE
Bring installation commands to a script file [14805]

### PR DESCRIPTION
**Pull request by richiware**

### CODE REVIEW

File: docs/resources/scripts/linux_binary_installation.bash [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+set -e
+
+##LINUX_BINARY_LOCALE
+locale  # check for UTF-8
+
+sudo apt update && sudo apt install -y locales
+# Any UTF-8 locale will work. Using en_US as an example
+sudo locale-gen en_US en_US.UTF-8
+sudo update-locale LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8
+export LANG=en_US.UTF-8
```
Comment (jsantiago-eProsima) [inline docs/resources/scripts/linux_binary_installation.bash:12]: Wouldn't this overwrite the user's current locale? Maybe the script should skip this block if a UTF-8 locale is detected.

File: docs/resources/scripts/linux_source_installation.bash [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+
+##LINUX_SOURCE_LOCALE
+locale  # check for UTF-8
+
+sudo apt update && sudo apt install -y locales
+# Any UTF-8 locale will work. Using en_US as an example
+sudo locale-gen en_US en_US.UTF-8
+sudo update-locale LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8
+export LANG=en_US.UTF-8
```
Comment (jsantiago-eProsima) [inline docs/resources/scripts/linux_source_installation.bash:13]: Same comment as above.

File: docs/resources/scripts/linux_binary_installation.bash (hunk 1 above)
Comment (richiware) [reply docs/resources/scripts/linux_binary_installation.bash:19]: Done in e9d392d

**Review by jsantiago-eProsima — APPROVED** [review]
LGTM

**Pull request merged**